include/inviwo/core/datastructures/camera.h

```diff
@@ -204,44 +207,43 @@ bool operator!=(const PerspectiveCamera& lhs, const PerspectiveCamera& rhs);
  * @see Camera
  * @see OrthographicCamera
  */
-class IVW_CORE_API OrthographicCamera : public Camera {
+class IVW_CORE_API OrthographicCamera final : public Camera {
 public:
     OrthographicCamera(vec3 lookFrom = vec3(0.0f, 0.0f, 2.0f), vec3 lookTo = vec3(0.0f),
                        vec3 lookUp = vec3(0.0f, 1.0f, 0.0f), float nearPlane = 0.01f,
-                       float farPlane = 10000.0f, vec4 frustum = vec4(-01, 10, -10, 10));
+                       float farPlane = 10000.0f, float width = 10.f, float aspectRatio = 1.f);
```

Comment:
Does it not make more sense to specify width, height instead of width, aspectRatio?

Reply:
nope, since width is the `width` of the orthographic camera frustum in world space and the aspect ratio comes from the canvas/viewport. Two entirely different things!

Reply:
Hm, the projection matrix calculation seem to use width/height derived from the aspect ratio:
    const float halfHeight = halfWidth / aspectRatio_;

Reply:
yes, usually we want the aspect of the frustum to be the same as that for the canvas.

Reply:
I see, I guess it is much easier to deal with the aspect ratio than to change the height. We might want to consider adding some explanation to the parameters.

Reply:
agree with that